includes/class-fs-plugin-updater.php

```diff
@@ -183,15 +245,41 @@ function edit_and_echo_plugin_update_row( $file, $plugin_data ) {
 
             $r = $current->response[ $file ];
 
-            $plugin_update_row = preg_replace(
-                '/(\<div.+>)(.+)(\<a.+\<a.+)\<\/div\>/is',
-                '$1 $2 ' . sprintf(
-                    $this->_fs->get_text_inline( '%sRenew your license now%s to access version %s security & feature updates, and support.', 'renew-license-now' ),
-                    '<a href="' . $this->_fs->pricing_url() . '">', '</a>',
-                    $r->new_version ) .
-                '$4',
-                $plugin_update_row
-            );
+            if ( ! $this->_fs->has_any_active_valid_license() ) {
+                $plugin_update_row = preg_replace(

```

Comment:
@fajardoleo same here, please add an explanation of what this regex does. If possible, include an example of an input and output.